### DISCUSSION TITLE
How to keep neogit opened after pressing `<cr>` on file?

### DISCUSSION THREAD

**Discussion by fjchen7** [NeogitOrg/neogit#912] [Q&A]
Neogit will close after I go to file by `<cr>`. How to make it remain opened?

**Comment by CKolkey** [comment]
You could also open the file in a split? I'm not sure I understand the question. Maybe you could make a video of the behaviour?

**Reply by muhajirdev** [reply]
How do I do that @CKolkey ?

**Comment by fjchen7** [comment]
https://github.com/NeogitOrg/neogit/assets/10106636/830226b5-ff2e-4a6a-b5de-500518525d07

The neogit page closes once I go to file by keymap `<cr>`. I would like to keep it remain opened.

**Reply by mangelozzi** [reply]
I think what the OP is saying if you open Neogit in a new tab, then you select a file to view it (I also often use Neogit to swtich between buffers I am working on), then he wish to return to neogit. But neogit will be closed. If neogit is open, you can `:tabp` to go back to the tab you were on and neogit will stay open, but if you select a file in neogit, then the neogit tab is closed (this is the scenario I think the OP has the issue with).

Currently I think one can open a vertical/horizontal split if one wishes to view a file a return. This is great if you have a big screen, if you dont, then it would be nice to resume neogit where you left off.

**Comment by muhajirdev** [comment]
facing the same issue, any progress here?

**Reply by CKolkey** [reply]
yes, should be released in the next week :)

**Reply by CKolkey** [reply]
Released

**Comment by ranjithshegde** [comment]
Came here looking for the same. I always open Neogit with kind='split' so that I have status on the bottom and file on the top. But when I go to Neogit window and select a different file, it autocloses Neogit status window. I want to keep that open. Is this a possibility why settings that I am not seeing? auto_close set to false does not change this behaviour

**Reply by rbhanot4739** [reply]
@CKolkey Even I am continuing to see this behaviour, is there a flag/option to toggle the auto closing of Neogit window ? The other slightly more nuanced behaviour I observed was that on hitting the `<cr>` it takes you to the beginning of the file and ideally it should take you to the last position in that file. I know this might not be vert straight forward but just wanted to flag it here .

**Comment by chuan29812** [comment]
```
      mappings = {
        status = {
          ['<cr>'] = 'TabOpen',
        },
      },
```

this is currently what im using to get around that problem
